spec/models/champ_spec.rb

```diff
@@ -453,4 +453,15 @@
       expect(champ_text_row_1.dossier_id).to eq(champ.dossier_id)
     end
   end
+
+  describe "#valid?" do
```

Comment:
Pinaillage : généralement j'ai l'impression qu'on met les tests de validation dans un bloc 

```ruby
describe "validations" do
  # …
end
```

plutôt vers le haut du fichier de specs.

Reply:
C'est noté :) Et modifié